src/client/themes/index.ts

```diff
@@ -21,6 +21,8 @@ export const getTheme = (theme?: Theme): DefaultTheme => {
     case 'explorer':
       return explorerTheme;
     default:
+      const isDarkMode: MediaQueryList = window?.matchMedia('(prefers-color-scheme: dark)');
+      if (isDarkMode.matches) return darkTheme;
       return lightTheme;
```

Comment:
nit
```suggestion
      return isDarkMode.matches ? darkTheme : lightTheme;
```

Reply:
or
```suggestion
      const { matches: prefersColorSchemeDark } = window?.matchMedia('(prefers-color-scheme: dark)');
      return prefersColorSchemeDark ? darkTheme : lightTheme;
```
so the variables names are more in line with what the code is doing